### PR TITLE
Dismissible notification functionality added

### PR DESCRIPTION
**Pull request by andrewcouling**
New icons and styling added to all themes.
Messages now stored in an array before being printed.
Assistance from @timschofield and @TurboPT  .

### CODE REVIEW

File: css/WEBootstrap/default.css [hunk 1]
```diff
@@ -184,18 +184,47 @@ tr.striped_row:nth-of-type(even) {
...
+    float: right;
+    font-size: 20px;
+	line-height: 17px;
+	cursor: pointer;
+    transition: 0.3s;
+}
+.MessageCloseButton:hover {
+    color: #606060;
+}
+#MessageContainerFoot {
```
Comment (TurboPT) [inline css/WEBootstrap/default.css]: @andrewcouling : in all the other CSS files, this style only has one property?

Comment (andrewcouling) [reply css/WEBootstrap/default.css]: This was intentional. Something about the layout of the bootstrap theme was causing the ends of the message boxes to be clipped.  So I just reduced the width by 5%.

Comment (TurboPT) [reply css/WEBootstrap/default.css]: Ok, no problem on that, just thought I'd ask. Figured that there might be some reason.

**Comment by andrewcouling** [comment]
Thanks @TurboPT .
Implementing the new message icons and style across all themes did feel like a lot of duplication, but I also felt that there was some benefit in allowing future modifications to style at theme level.
Please continue to challenge my approach to these sorts things. Its always appreciated.

**Comment by TurboPT** [comment]
@andrewcouling 
Yes, the per-theme-level would allow maximum flexibility and customization, I only mentioned the opposite of having one standard way of messaging app wide, to see if it might spark any other community input.

No worries, I believe that we are on the same page, just differing ideas. I'm good either way.

**Pull request merged**